modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition_manager.py

```diff
@@ -51,6 +51,9 @@ def get_objects_from_partitions(cls, partitions):
         list
             The objects wrapped by `partitions`.
         """
+        for idx, part in enumerate(partitions):
+            if hasattr(part, "force_materialization"):
+                partitions[idx] = part.force_materialization()
```

Comment:
Why do we need to materialize an axis partition into a single partition? Can't we use [`.unwrap()`](https://github.com/modin-project/modin/blob/317bd75d0677c0ae1a400f7313953ee6638185ac/modin/core/dataframe/base/partitioning/axis_partition.py#L131) here?

Reply:
I do not understand why. This will return the partitions and dataframes in them, which will still have to be combined into one.

Reply:
Discussed offline. Now, I understand why, but this will increase the complexity of the code. While this is not the bottleneck right now, I propose to postpone the solution of this issue.

Reply:
https://github.com/modin-project/modin/issues/5269